packages/dynamite/dynamite_runtime/lib/src/dynamite_client.dart

```diff
@@ -383,9 +383,10 @@ class DynamiteClient {
     Uri baseURL, {
     this.baseHeaders,
     http.Client? httpClient,
-    this.cookieJar,
+    CookieJar? cookieJar,
     this.authentications,
   })  : httpClient = httpClient ?? http.Client(),
+        cookieJar = cookieJar ?? PersistCookieJar(),
```

Comment:
I would prefer using a memory-only cookie jar by default. In the worst case something will not work between runs, but with a persistent cookie jar the worst case would be authenticating with the wrong user and executing (unintentionally) harmful requests.

Reply:
Why? We have to consider that this is going to be the default for all dynamite generated libraries.
I don't fully understand the reasoning why an in memory cache is less dangerous than a file backed implementation or how this would affect the general nextcloud package

Reply:
Storing the cookies on disk makes it possible to read them easily so attackers could do that, but I was more thinking about leaking cookies where it shouldn't happen, like in our case with different accounts on the same server.

Reply:
Should we default to the `WebCookieStore` which does not store any cookies at all.

Reply:
Instead of that I would make the cookie jar nullable like it was and provide no implementation.
This would be my preferred solution as we can't anything much wrong by default.

Reply:
So just what we have on main minus re-exporting the package.